muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/current-student/filters/graph-filter.tsx

```diff
@@ -0,0 +1,65 @@
+import {i18nType} from "~/reducers/base/i18n";
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {StateType} from '~/reducers';
+import Dropdown from '~/components/general/dropdown';
+
+interface GraphFilterProps {
+  i18n: i18nType,
+  graphs: string[],
+  filteredGraphs: string[],
+  handler: any,
+  modificator?: string
+}
+
+class GraphFilter extends React.Component<GraphFilterProps> {
+  constructor(props:GraphFilterProps) {
+    super(props);
+  }
+  
+  dropdownFilter(){
+    let modificator = this.props.modificator || "";
+    if (modificator == "-dropdown-only" || modificator == ""){
+      return <Dropdown modifier={"graph-filter" + modificator} persistant={true} items={this.props.graphs.map((graph)=>{
```

Comment:
Very minor thing :) it's totally okay but you can actually just do `persistant` without `persistant={true}` and it's exactly the same; but it's okay, you don't have to change it, it's just the way it's commonly done.